infrastructure/modules/keyvault/main.tf

```diff
@@ -0,0 +1,71 @@
+data "azurerm_client_config" "current" {}
+
+resource "azurerm_key_vault" "secrets_vault" {
+  name                = var.name
+  location            = var.location
+  resource_group_name = var.resource_group_name
+  tenant_id           = data.azurerm_client_config.current.tenant_id
+  soft_delete_enabled = true
+
+  sku_name = "standard"
+
+  network_acls {
+    default_action = var.network_acls_default_action
+    bypass         = "AzureServices"
+    ip_rules       = var.access_CIDR
+  }
```

Comment:
so for prod, we're using this to restrict on IP? do we need that as well as the access_policy below?

Reply:
Yeah - we need both. This rule is specific for non-portal (API) access, which we want to lock down to MHRA IP's in line with pen testing suggestion. The below access policies refer to what each of us has permissions to do, and represents our current configuration that we set-up through the portal.